### ISSUE TITLE
Install VIA ventoy

### ISSUE THREAD

**Issue by forsakensingularity** [SpiralLinux/SpiralLinux-project#314]
Not  a Bug - just a note: 

I was using Ventoy 1.0.95 and it does allow EFI install.

Just an FYI.   I have a frankenstien laptop with many HardDrives attached - all SATA, and the installer wants to place EFI on sda1, which is fine - I use sda[*] mostly as  junk NTFS drive and not as an OS type drive.

SpiralLinux XFCE is fast on my old i5 HP Laptop  :)

**Comment by Anonymo** [comment]
https://github.com/tpowa/Archboot/commit/0495c926f90a2fa5d364cac2d335491e6de7232f  Not sure if this is helpful at all but Archboot dev added Ventoy support recently.

https://github.com/tpowa/Archboot/commit/85c9b09aac249637b49fbd9d051fb72c175ee78c

**Comment by geckolinux** [comment]
Thanks for the comments. It may nominally "work", but anything that uses non-standard methods to create the USB device is likely to cause weird issues that are hard to debug, because it might change some of the live boot parameters that are needed for the system to boot and configure itself correctly for the live session.

**Comment by forsakensingularity** [comment]
But nothing is really changed.  I just download the ISO and drop into Ventoy area and pick the ISO I want to use that day.  I dont 'burn' any USBs ... My internal HDD is not even formatted ... it is why I can test so many spins, mainly because my backups arent tied to my system  - so I can change OSes anytime I want.  I did actually install all the latest SpiralLinux verions and choose Cinnamon as the default Ventoy boot image tho...I just reformatted the drive after each test was completed.

I dont trust anyone  - so because of that my Installations rarely survive longer than a day or two and I switch among three or four systems each week.

I am prolly insane, but I wont use Linux any other way  :/ lol
